ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -296,14 +296,33 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
 static NSString * const ORKInstruction0StepIdentifier = @"instruction";
 static NSString * const ORKInstruction1StepIdentifier = @"instruction1";
 static NSString * const ORKCountdownStepIdentifier = @"countdown";
+static NSString * const ORKCountdownAudioRecorderIdentifier = @"countdown.audio";
 static NSString * const ORKAudioStepIdentifier = @"audio";
+static NSString * const ORKAudioAudioRecorderIdentifier = @"audio.audio";
 static NSString * const ORKTappingStepIdentifier = @"tapping";
+static NSString * const ORKTappingAccelerometerRecorderIdentifier = @"tapping.accelerometer";
 static NSString * const ORKConclusionStepIdentifier = @"conclusion";
 static NSString * const ORKFitnessWalkStepIdentifier = @"fitness.walk";
+static NSString * const ORKFitnessWalkPedometerRecorderIdentifier = @"fitness.walk.pedometer";
+static NSString * const ORKFitnessWalkAccelerometerRecorderIdentifier = @"fitness.walk.accelerometer";
+static NSString * const ORKFitnessWalkDeviceMotionRecorderIdentifier = @"fitness.walk.deviceMotion";
+static NSString * const ORKFitnessWalkLocationRecorderIdentifier = @"fitness.walk.location";
```

Comment:
In my opinion, and I'm open to counter-argument, the identifiers here do not need prefixes (e.g. "fitness.walk."). We don't use prefixes on step identifiers, so there should be no need to do so at the next level of the hierarchy.

The hierarchy itself gives this context, and uniqueness is manually preserved because we only include one recorder of each type in a given step.

I would be in favor of having just one identifier name used for each type of recorder for these predefined recorders.